packages/runtime/package.json

```diff
@@ -177,8 +177,6 @@
     "path-to-regexp": "^6.2.1",
     "polished": "3.0.3",
     "react-player": "^2.16.0",
-    "redux": "^4.0.5",
-    "redux-thunk": "^2.3.0",
```

Comment:
`redux` and `redux-thunk` come bundled with `@reduxjs/toolkit`, so we don't need to explicitly import them.